druid/Cargo.toml

```diff
@@ -33,6 +33,10 @@ fnv = "1.0.3"
 xi-unicode = "0.2.0"
 image = {version = "0.23.2", optional = true}
 
+[dev-dependencies]
+tempfile = "3.1.0"
+piet-common = {version = "0.0.11", features = ["png"]}
```

Comment:
I only need this if i want to run the druid tests in the druid folder.
I think its good to be able to do that, you can atm. But i dont like that this is defined in two places. @xStrom can you think of a good way to do this other than putting the feature in to the normal deps of `druid-shell/Cargo.toml` how i had it before your comment?

Reply:
Maybe feature gating it would allow for a single definition. However I think having this defined in two places is fine. That's nothing compared to what is going on inside piet itself.

Reply:
Ok, I'm not sure that the fact its a mess somewhere else is a great argument for it being bad but if were happy with it like this for now I will take another look at this when I do the PR to expose testing as public behind a feature.

Thanks @xStrom

Reply:
To quote @cmyr *we're allowed to be more hacky with testing code*.